src/cloud/internal/cloudservice.cpp

```diff
@@ -41,6 +42,17 @@ static const QString REFRESH_TOKEN_KEY("refresh_token");
 static const QString DEVICE_ID_KEY("device_id");
 
 constexpr int USER_UNAUTHORIZED_ERR_CODE = 401;
+constexpr int INVALID_SCORE_ID = 0;
+
+int scoreIdFromSourceUrl(const QUrl& sourceUrl)
```

Comment:
Better add `static`, else this function will be global

Reply:
fixed